lte/gateway/python/integ_tests/s1aptests/s1ap_utils.py

```diff
@@ -1196,7 +1225,10 @@ def restart_sctpd(self):
         """
         Restart sctpd service explicitly because it is not managed by magmad
         """
-        self.exec_command("sudo service sctpd restart")
+        if self._init_system == InitMode.SYSTEMD:
+            self.exec_command("sudo service sctpd restart")
+        elif self._init_system == InitMode.DOCKER:
+            self.exec_command("sudo docker restart sctpd")
```

Comment:
why is `sudo` needed here and not for the `docker-compose` restart?

Reply:
Good point, thanks! I've removed the `sudo` here.